src/utils/panes/index.tsx

```diff
@@ -32,25 +33,29 @@ const PaneContext = createContext<PaneContextData>({
 
 type PaneProviderProps = {
   children: ReactNode;
+  collapsed: boolean;
 };
 
-const useStyles = makeStyles({
+const useStyles = makeStyles<Theme, { collapsed: boolean }>(() => ({
   container: {
     bottom: 16,
-    position: 'absolute',
+    position: 'fixed',
     right: 16,
-    top: 16,
+    top: ({ collapsed }) => (collapsed ? '17vh' : '35vh'),
```

Comment:
Two problems here:

* First, the position can't be _relative_ (`vh`), it must be absolute to reflect the actual height of the header
* Second, the header can have different height, not just two states (collapsed/expanded) but also depending on how much content is in the header, whether it has tabs etc.